openassessment/xblock/test/test_peer.py

```diff
@@ -286,7 +286,6 @@ def test_turbo_grading(self, xblock):
         self.assertIsNotNone(peer_response)
         self.assertNotIn(submission["answer"]["parts"][0]["text"].encode('utf-8'), peer_response.body)
         self.assertNotIn(submission["answer"]["parts"][1]["text"].encode('utf-8'), peer_response.body)
-        self.assertIn("Peer Assessments Complete", peer_response.body)
```

Comment:
Perhaps instead look for "You have successfully completed"? I imagine the desire was to make sure the correct template was being returned.